internal/importer/importer.go

```diff
@@ -76,7 +78,7 @@ func (i *azureImporter) InitWritesProblem(token wharfapi.Token, provider wharfap
 	}
 	log.Debug().
 		WithUint("id", i.provider.ProviderID).
-		WithString("name", i.provider.Name).
+		WithString("name", string(i.provider.Name)).
```

Comment:
Why this name cast?

Reply:
Because `i.provider.Name` is of type `ProviderName`, defined as `type ProviderName string`.